sdk/keyvault/Azure.Security.KeyVault.Secrets/tests/Azure.Security.Keyvault.Secrets.Tests.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>This is the Microsoft Azure Key Vault Secrets client test library</Description>
+    <AssemblyTitle>Microsoft Azure.Security.KeyVault.Secrets client test library</AssemblyTitle>
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <PackageTags>Microsoft Azure Key Vault Secrets Tests</PackageTags> 
+    <PackageReleaseNotes>
+      <![CDATA[
+      ]]>
+    </PackageReleaseNotes>
+
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <NoWarn>$(NoWarn);3021</NoWarn>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+
+    <LangVersion>8.0</LangVersion>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <IsPackable>false</IsPackable>
+    <GenerateProgramFile>false</GenerateProgramFile>
```

Comment:
Apart from `GenerateProgramFile` doesn't look like we need these properties.
Also, `TargetFramework` is already defined in line 13 following the targets of the repo.  Do we really need 2.2 for the tests?